enclave/sgx/eeid_attester.c

```diff
@@ -131,17 +139,30 @@ static oe_result_t _eeid_get_evidence(
     if (endorsements_buffer_size)
         *endorsements_buffer_size = 0;
 
-    if (!eeid || eeid->data_size == 0 ||
-        eeid->signature_size != sizeof(sgx_sigstruct_t))
+    if (eeid->signature_size != sizeof(sgx_sigstruct_t))
         OE_RAISE(OE_FAILURE);
 
     size_t eeid_size = oe_eeid_byte_size(eeid);
 
+    // Serialize custom claims, including EEID base image sizes
```

Comment:
Base enclave size_setting info is not needed by the eeid enclave attestation verifier to do its verification work. See comments in eeid_verifier.c. A base enclave size_setting claim is not useful for the Relying Party either, as the potenitally malicious additional content loaded into an EEID enclave might be able to modify code logic to spoof base enclave size_setting info in the attestaiton evidence, if base enclave size_setting info is critical to detect the malicious cotent. I don't think it needs to be inlcuded in the attestation evidence.